Source/Core/Common/StringUtil.h

```diff
@@ -13,6 +13,8 @@
 
 #include "Common/Common.h"
 
+std::string StringFromFormat(const char* format, va_list args);
```

Comment:
This is a dangerous; on some platforms, va_list is a void* or char*, so you could end up calling the wrong overload.  Please change the name.